opennebula/defaults.yaml

```diff
@@ -2,50 +2,42 @@ kernel:
   Linux:
     oneadmin:
       home: /var/lib/one
-      optional_groups:
-        - kvm
-        - libvirt
-        - ssl-cert
       config:
         manage:
-          - ec2_auth
-          - occi_auth
+#          - ec2_auth
```

Comment:
OK, last note is already so long, but I there is still something to say:
Managing
- ec2_auth
- occi_auth
- oneflow_auth
- onegate_auth
- sunstone_auth
on initial run will not work (I described it in the comment of the commit), because oned complains about existing _auth-files before the database is bootstrapped.
Therefor I suggest disabling managing these files in the default.yaml, because installation will fail otherwise.
If somebody is knowing what he's doing he can override it in the pillar data.